caliper_sender.py

```diff
@@ -0,0 +1,127 @@
+import caliper
+import psycopg2
+# from dotenv import load_dotenv
+import requests, json, sys, os, logging
+from datetime import datetime, date, time
+
+import os
+
+# load_dotenv()
+
+def send_caliper_event():
+    print("Connect to database...")
+    conn = psycopg2.connect(
+        dbname = "runestone",
+        user = os.getenv("USER"),
+        password = os.getenv("PASSWORD")
+        )
+    cur = conn.cursor()
+
+    # create a runtime table, get the last runtime
+    try:
+        cur.execute("CREATE TABLE IF NOT EXISTS execute_time (runtime timestamp NOT NULL, status varchar(10) NOT NULL);")
```

Comment:
No need to have the "create table" check during each send_caliper_event call

Reply:
Right, this "CREATE TABLE" should be moved out to a separate method.